codalab/rest/util.py

```diff
@@ -114,39 +120,40 @@ def get_bundle_infos(
                 if child_perms[child_uuid] >= GROUP_OBJECT_PERMISSION_READ
             ]
 
+    if get_single_host_worksheet:
+        # Get back a dict of bundle uuids -> list of worksheet uuids (max of 5)
```

Comment:
Slightly weird comment, it should explain the whole block instead of just the get_n_host_worksheet_uuids function right?  Would be better to explain the block logic (first get at most 5 host worksheet, filter with permission and ok to have no worksheet returned if none of 5 qualifies)